src/ConnectionPool.jl

```diff
@@ -640,14 +640,17 @@ function getconnection(::Type{TCPSocket},
                        host::AbstractString,
                        port::AbstractString;
                        keepalive::Bool=false,
-                       connect_timeout::Int=0,
+                       connecttimeout::Int=0,
```

Comment:
Even thought it's not documented, I think this needs to remain `connect_timeout` otherwise it's a breaking change.

Reply:
Makes sense, fixed.